OS/DiskOS/Programs/Disks/load.lua

```diff
@@ -1,39 +1,54 @@
+-- Load a game into memory either from a .png or .lk12 file or from the clipboard
+
 local source = select(1,...)
 
 local term = require("terminal")
 local eapi = require("Editors")
 
 local png = false
 
-if source and source ~= "@clip" and source ~= "-?" then
-  source = term.resolve(source)
-  if source:sub(-4,-1) == ".png" then
-    png = true
-  elseif source:sub(-5,-1) ~= ".lk12" then
-    local lksrc = source..".lk12"
-    if fs.exists(lksrc) then
-      source = lksrc
-    elseif fs.exists(source..".png") then
-      source = source..".png"
+local lk12Data = nil
+
+-- Load from clipboard
+if source == "@clip" then
+  lk12Data = clipboard()
+else
+  -- Use the last know path if no file was provided
```

Comment:
```suggestion
  -- Use the last known path if no file was provided
```
I tiny spelling fix.

Reply:
Fixed in commit 709acf097b4b45c902411402e334a0db4c24d65a.